moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch

```diff
@@ -89,7 +89,9 @@
 
     <param name="allow_trajectory_execution" value="$(arg allow_trajectory_execution)"/>
     <param name="sense_for_plan/max_safe_path_cost" value="$(arg max_safe_path_cost)"/>
-    <param name="jiggle_fraction" value="$(arg jiggle_fraction)" />
+    <param name="planning_pipelines/ompl/jiggle_fraction" value="$(arg jiggle_fraction)" />
+    <param name="planning_pipelines/chomp/jiggle_fraction" value="$(arg jiggle_fraction)" />
+    <param name="planning_pipelines/pilz_industrial_motion_planner/jiggle_fraction" value="$(arg jiggle_fraction)" />
```

Comment:
The only line I am unsure of is this one (`planning_pipelines/pilz_industrial_motion_planner/jiggle_fraction`). The parameter `jiggle_fraction` is read by the `FixStartStateCollision` planning adapter. CHOMP and OMPL both use that adapter, but pilz_industrial_motion_planner doesn't at the moment.

* Do we remove this line? The parameter isn't read, so it does nothing.
* Do we keep it in? Would be more consistent, and perhaps the user later decides to add the `FixStartStateCollision` adapter to the pilz pipeline.

Reply:
They don't belong into this file at all, but into their respective `<planner>_planning_pipeline` files.
I propose to simply drop the useless `jiggle_fraction` argument @davetcoleman introduced in 2013 and move them to their respective files with the same default value.
I pushed a respective commit.

Please have a look @mintar . Would be nice to get a quick second review from @felixvd . :sunglasses: